content/docs/features/querying.mdx

```diff
@@ -0,0 +1,161 @@
+
+```http
+GET https://api.ordercloud.io/v1/buyers/xyz/users?LastName=Smith|Jo*&FirstName=Johnny HTTP/1.1
+```
+
+This will return "John Smith", "Johnny Jones", and "John Johnson". Maybe you want "John Smith" but not "John Jones".
+
+### Negate and Logical AND
+
+You can negate your conditions by prefixing them with `!`, and logically `AND` them together by simply providing the same parameter multiple times.
+
+```http
+GET https://api.ordercloud.io/v1/buyers/xyz/users?LastName=!Smith&LastName=!Jones HTTP/1.1
+```
+
+This will return all users _except_ those with last name "Smith" or "Jones".
+
+### Comparison Operators
+
+Dates and numeric values support `>` (greater than) and `<` (less than) prefixes.
+
+```http
+GET https://api.ordercloud.io/v1/buyers/xyz/users?DateCreated=>2015-01-01 HTTP/1.1
+```
+
+For a more advanced example, let's say you want users whose ID is the range of 0 to 9 inclusive. Ranges are not directly supported, but you can use the existing features to achieve this.
+
+```http
+GET https://api.ordercloud.io/v1/buyers/xyz/users?ID=0|1|2|3|4|5|6|7|8|9 HTTP/1.1
+```
+
+Of course that's not going to be feasible if the range is very large or you're dealing with floating-point numbers. A better way would be to leverage `>` and `<`, but we want the range to be inclusive, and there are no `>=` or `<=` operators. We can however leverage the `!` operator.
+
+Here we're saying "give me all users whose ID is not less than 0 and not greater than 9", which is effectively equivalent to our 0-9 range.
+
+```http
+GET https://api.ordercloud.io/v1/buyers/xyz/users?ID=!<0&ID=!>9 HTTP/1.1
+```
+
+### Absence of a value
+
+Sometimes you need a list of items where some field either doesn't exist or is null or undefined. To search on the absence of a field you can combine the negate operator and the wildcard operator
+
+```http
+GET https://api.ordercloud.io/v1/orders?xp.Billed=!* HTTP/1.1
+```
+
+Here we're saying "give me all orders that haven't been billed (xp.Billed is undefined, null, or doesn't exist)"
+
+### Performance Considerations
+
+Your data is highly indexed for fast retrieval using any of the methods above, including deep XP object graphs. However, there are a few things to keep in mind to keep search and listing functionality efficient. Filters with many `OR` conditions or with values that start with the `*` wildcard character may be particularly performance-sensitive. The larger the set of data, the more likely you are to notice any performance impact. It pays to know the data model well, and use grouping constructs like User Groups and Categories effectively.
+
+## Sorting
+
+In addition to being able to hone in the data itself, you can also indicate to OrderCloud how you want it sorted. Perhaps you want to sort the results (in ascending order) by the last name of the person who placed an order. To do this you can simply include the `sortBy` parameter for the field you would like to sort by.
```

Comment:
"hone in" is used a lot, maybe  "narrow down" is a more common phrase to use?

Reply:
updated ⚡️